packages/block-editor/src/components/list-view/branch.js

```diff
@@ -82,7 +82,7 @@ const countReducer =
 function ListViewBranch( props ) {
 	const {
 		blocks,
-		selectBlock,
+		selectBlock = () => {},
```

Comment:
Does it make sense to declare this as a constant outside of the component, so it won't be redeclared on every re-render that doesn't have a `selectBlock` prop?

Similar to how it's done here:

https://github.com/WordPress/gutenberg/blob/f0fe8f64fe0afef0f383ffa8ea6320d1c774f600/packages/block-editor/src/components/block-alignment-matrix-control/index.js#L12-L17

Reply:
Sounds good I will push the update in few minutes.

Btw, I've seen both patterns used in the codebase. Example:

https://github.com/WordPress/gutenberg/blob/8c44ccb676110d43a04cf8080836d917c59aec62/packages/components/src/form-token-field/index.tsx#L52-L54

Reply:
True! I see those as potential optimizations we can do in the future.

Reply:
Updated in [8ca2a1c](https://github.com/WordPress/gutenberg/pull/46266/commits/8ca2a1caba8befd0cb251fe067f6d260dc5abefc).